src/AlgoliaSearch.js

```diff
@@ -550,12 +553,27 @@ AlgoliaSearch.prototype = {
       })
     };
 
+    var JSONPParams = map(postObj.requests, function prepareJSONPParams(request, requestId) {
+      return requestId + '=' +
+        encodeURIComponent(
+          '/1/indexes/' + request.indexName + '?' +
```

Comment:
Well, I understand that `encodeURIComponent` encodes the URI, but not oh does it handle encoding `'/1/indexes/toto?tata=1?queryString=tutu'`. 
I'm just curious about the sides effects